synfig-studio/src/gui/resources/css/synfig.mac.css

```diff
@@ -7,3 +7,8 @@
 	box-shadow: none;
 	margin    : 0;
 }
+
+#parameters_panel {
+    font-size: 11pt;
+    font-weight: 300;
```

Comment:
@ice0 please use `tab` for indentation here too (Codacy is complaining and the rest of CSS file use it indeed)